.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+   "python.formatting.provider": "black"
```

Comment:
Editor settings do not need to be included in the Project. Please, remove this file from the commit.